packages/cli/src/helpers/getUnappliedSchemaOverridesTableNames.ts

```diff
@@ -0,0 +1,29 @@
+import { isObject } from './isValidSchemaDefOverrides';
+
+export function getUnappliedSchemaOverridesTableNames(
```

Comment:
remember what I said about co-location: this function is in a "helpers" folder, but it will only ever be used in a single place. So better to co-locate it.

Also in general, try to avoid "helper" or "util" folders. Some things are actually helper/utils but quite often it's just a lazy way to drop unrelated code. So helper folders become a noisy bag of unrelated functionality over time.